www/getting-started.rst

```diff
@@ -2,151 +2,80 @@
 Getting Started
 ===============
 
-This page is intended to help the beginner get a handle on SciPy and be
-productive with it as fast as possible.
+Got the SciPy packages installed? Wondering what to do next? 
 
-What are NumPy, SciPy, matplotlib, ...?
----------------------------------------
+"Scientific Python" doesn't exist without "Python". SciPy skills need to build on a foundation of standard programming skills. While Python itself has an `official tutorial <https://docs.python.org/3/tutorial/>`_ , countless resources exist online, in hard copy, in person, or whatever format you prefer.
 
-SciPy and friends can be used for a variety of tasks:
+Just remember to have fun, make mistakes, and persevere. 
 
-* *NumPy*'s array type augments the Python language with an efficient data
-  structure useful for numerical work, e.g., manipulating matrices.  *NumPy*
-  also provides basic numerical routines, such as tools for finding
-  eigenvectors.
+Where to Write
+==============
 
-* *SciPy* contains additional routines needed in scientific work: for example,
-  routines for computing integrals numerically, solving differential equations,
-  optimization, and sparse matrices.
+`Jupyter <https://jupyter.org/>`_ notebooks combine code, markdown, and more in an interactive setting. They are an excellent tool for learning, collaborating, experimenting, or documenting. Notebooks can run on your local machine, and `MyBinder <https://mybinder.org/>`_ also serves Jupyter notebooks to the browser without the need for anything on the local computer. For example, `MyBinder Elegant Scipy <https://mybinder.org/v2/gh/elegant-scipy/notebooks/master?filepath=index.ipynb>`_ provides an interactive tutorial.
 
-* The matplotlib_ module produces high quality plots. With it you can turn your
-  data or your models into figures for presentations or articles. No need to do
-  the numerical work in one program, save the data, and plot it with another
-  program.
+Jupyter runs by calling to `IPython <https://ipython.org/>`_ behind the scenes, but IPython itself also acts as a standalone tool. A *command-line* of individual statements and returned values, IPython is useful for debugging and experimenting.
 
-* Using IPython_ makes interactive work easy. Data processing, exploration of
-  numerical models, trying out operations on-the-fly allows to go quickly from
-  an idea to a result. See the `IPython site <http://ipython.org/>`__ for many
-  examples.
+Editors for scripts run from minimal, like Window's Notepad or `Emacs <https://www.gnu.org/software/emacs/>`_ , to the fully-featured and customizable, like `Atom <https://atom.io/>`_ or `Visual Studio Code <https://code.visualstudio.com/>`_ . Jupyter and IPython work well for interacting with code and experimenting, but scripts, files with a `*.py` extension, form backends, infrastructure, or anything heavy-duty.  
```

Comment:
It would be nice to expand this a little. If Jupyter is _"an excellent tool for learning, collaborating, experimenting, or documenting"_, then editors or IDEs are ... (e.g. "the best tools to use for writing analysis scripts, reproducible and reusable code, and Python packages and libraries").

Reply:
Rewrote this section.